shaders/MucocystEffect.gdshader

```diff
@@ -0,0 +1,36 @@
+shader_type spatial;
+render_mode unshaded;
+
+// Any changes to these two must be matched in Membrane.gdshader
+// Set to 0 to disable wiggle
+uniform float wigglyNess = 1.f;
+uniform float movementWigglyNess = 1.f;
+
+uniform float fade = 0.25f;
+uniform vec3 tint : source_color = vec3(0.40, 0.80, 0.60);
+
+void vertex(){
+    vec3 worldVertex = (MODEL_MATRIX * vec4(VERTEX, 1.0)).xyz;
+    float size = length(VERTEX);
+
+	//worldVertex /= 2.f;
```

Comment:
Unless Membrane.gdshader has these same commented out lines, these should be removed. Also the comment near the top says the positioning math must match exactly the one in Membrane otherwise this effect won't match the shape of the membrane, so double checking that this math matches would be good.